src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,206 @@
+
+import iosConfirmSignIn from "/src/fragments/lib/auth/ios/totp/confirm_sign_in.mdx";
+
+<Fragments fragments={{ios: iosConfirmSignIn}} />
+
+### Enabling TOTP after a user is signed in
+
+TOTP MFA can be set up after a user has signed in. This can be done when the following conditions are met:
+
+- MFA is marked as **Optional** or **Required** in Cognito User Pool.
+- TOTP is marked as an enabled MFA method in Cognito user pool
+
+TOTP setup can be achieved by calling `setUpTOTP` and `verifyTOTPSetup` APIs that are available in the `Auth` category.
+
+Invoke the `setUpTOTP` API to generate a `TOTPSetupDetails` object that contains the `sharedSecret` which will be used to either to generate a QR code or can be manually entered into an Authenticator app.
+
+**`setUpTOTP` API**
+
+import iosSetUpTOTP from "/src/fragments/lib/auth/ios/totp/set_up_totp.mdx";
+
+<Fragments fragments={{ios: iosSetUpTOTP}} />
+
+Once the authenticator app is set up, the user can get a TOTP code that is generated by the Authenticator app and can provide it to the library to using the `verifyTOTPSetup` API to complete the TOTP setup process.
+
+**`verifyTOTPSetup` API**
+
+import iosVerifyTOTPSetup from "/src/fragments/lib/auth/ios/totp/verify_totp_setup.mdx";
+
+<Fragments fragments={{ios: iosVerifyTOTPSetup}} />
+
+### Recovering from a lost TOTP device
+
+If a user loses access to their TOTP device, they would need to contact an administrator to help get access to their account. Based on the Cognito User Pool configuration, the administrator can use the [AdminSetUserMFAPreference](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminSetUserMFAPreference.html) to either change the MFA preference to a different MFA method or to disable MFA for the user.
+
+<Callout warning>
+
+In a scenario where MFA is marked as **Required** in Cognito User Pool and another MFA method is not set up, the administrator would need to first initiate an [AdminUpdateUserAttributes](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUpdateUserAttributes.html) call and update the user’s phone number attribute. Once this is complete, the administrator can continue changing the MFA preference to SMS as suggested above.
+
+</Callout>
+
+## Setting up user preferred MFA option
+
+You can use Time-based One-Time Password (TOTP) for multi-factor authentication (MFA) in your web or mobile applications. Amplify's Auth category includes support for handling TOTP setup and codes, offering an integrated solution for enhanced security for your users. Authenticator apps generate the temporary, one-time passwords. These apps, such as Google Authenticator, Microsoft Authenticator, have the TOTP algorithm built-in and work by using a shared secret key and the current time to generate a short-lived password.
```

Comment:
Is this paragraph needed here? Probably only if it's going to be its own page, but TOTP is well-established by this point on this page.

Reply:
I agree. Moved some of it to the TOTP section.